ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -1286,15 +1286,7 @@ private boolean isDeleteTombstoneRemovable(IndexEntry deleteIndexEntry, IndexSeg
       // TTL update + DELETE -> DELETE
       // PUT + TTL update -> PUT (the one relevant to this comment)
       // PUT + TTL update + DELETE -> DELETE
-      // TODO: move this blob store stats
-      Offset startOffsetOfLastIndexSegmentForDeleteCheck = getStartOffsetOfLastIndexSegmentForDeleteCheck();
```

Comment:
Add a to-do that we can remove `getStartOffsetOfLastIndexSegmentForDeleteCheck()` method once we verified the correctness.

Reply:
Will remove it.